news/2021-11-04-skinners-bulletin-august-2021.md

```diff
@@ -0,0 +1,90 @@
+
+<iframe width="100%" height="400" src="https://www.youtube-nocookie.com/embed/l0L2Zroknso" title="YouTube video player" frameborder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+### Kronii no Jikan
+
+Here we are with another detail-oriented [thetasigma](https://osu.ppy.sh/users/6234482) classic: [Kronii no Jikan](https://osu.ppy.sh/community/forums/topics/1406817)! Filled with clockwork from head to toe, this skin revolves around the popular VTuber Ouro Kronii, who is canonically known to be the Warden of Time. As much as thetasigma opted to create an anime-themed skin, there is another version that is entirely Kronii-free that you can try. Nevertheless, both versions immediately give out the impression that the skin certainly deserves recognition.
+
+Embracing antiquity very well, Kronii no Jikan is dominated by golden yellow and navy blue, denoting an elegant look on the skin. We encounter a rare layout choice for the menu buttons on the bottom left of the song selection menu, which looks utterly fantastic. The way selected game modes are denoted by highlighting their respective letters, the way a clock is used in place of the game modifier menu icon⁠ — every detail is noteworthy and shows us thetasigma's mastery. Another point reflecting the quality of the skin is, without a doubt, the amount of flavour texts scattered everywhere, especially the one located beneath the `menu-back` button. Cool stuff!
+
+Moving onto the gameplay interface, we see every aspect of the main menu reflected beautifully here too. However, this time gear wheels dominate the screen as if we are inside a typical clock. One of the most fabulous parts of the gameplay section is undoubtedly the pause menu, where menu options surround a cracked piece of clockwork, and the fail menu, a copy of the pause menu but with more cracks. The section pass/fail displays are another aspect of the skin dominated by gear wheels and Kronii herself, in addition to the skip button, which is centralised, making it look even more remarkable.
+
+In short, we highly recommend you take this *timeless* creation on a test ride! Thanks to the showcase video below, time constraints are no longer an issue:
+
+<iframe width="100%" height="400" src="https://www.youtube-nocookie.com/embed/NvVeoYrZMUo" title="YouTube video player" frameborder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+---
+
+And with that, it is time to conclude this issue of our bulletin! We're sincerely sorry about the unexpected delay on our side.
+
+We'll be moving things to a seasonal schedule from now on, meaning that the next issue of Skinners' Bulletin will be released in December. This will mean more stats, more skins, news and tips for you in neat three month blocks, so make sure to catch us then.
```

Comment:
```suggestion
We'll be moving things to a seasonal schedule from now on, meaning that the next issue of Skinners' Bulletin will be released in December. This will mean more stats, more skins, news and tips for you in neat three-month blocks, so make sure to catch us then.
```